incubator/spring-cloud-data-flow/values.yaml

```diff
@@ -0,0 +1,48 @@
+# Default values for spring-cloud-data-flow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+dataflowUsername: user
+dataflowPassword: password
+dataflowRoles: ROLE_VIEW, ROLE_CREATE
+dataflowAdminUsername: admin
+dataflowAdminPassword: admin
+dataflowAdminRoles: ROLE_MANAGE, ROLE_VIEW
+server:
+  image: springcloud/spring-cloud-dataflow-server-kubernetes
+  version: 1.2.2.RELEASE
+  imagePullPolicy: IfNotPresent
+  service:
+    type: LoadBalancer
+    externalPort: 80
+    internalPort: 80
+  resources:
```

Comment:
Don't specify defaults for resources. We've started recommending to leave this a conscious choice of the user. It should also be possible to test charts on systems with little resources, such as Minikube.

Do it like so:

```yaml
  resources: {}
  #  limits:
  #    cpu: 1.0
  #    memory: 2048Mi
  #  requests:
  #    cpu: 0.5
  #    memory: 1024Mi
```